### PR TITLE
JR2-253 Redesign Job Details page

### PR DESCRIPTION
**Pull request by Invizible**

### CODE REVIEW

File: src/main/webapp/i18n/de/job-detail.json [hunk 1]
```diff
@@ -37,6 +37,7 @@
         "language.level.GOOD": "Gut",
         "language.level.NONE": "keine Kentnisse",
         "language.level.VERY_GOOD": "Sehr Gut",
+        "language.level.PROFICIENT": "Professionell",
```
Comment (birom) [inline src/main/webapp/i18n/de/job-detail.json]: Please change the translations.csv file too. For more info: https://github.com/alv-ch/jobroom2#export--import-translation-json-files-to--from-csv-file. I've also modified that in this [PR](https://github.com/alv-ch/jobroom2/pull/104).
Thank you!

Comment (michael-wirth) [reply src/main/webapp/i18n/de/job-detail.json]: Yes, please change the translation.csv and generate the json file with the grade target csv2json.

The levels have been renamed. There are now `NONE, BASIC, INTERMEDIATE, PROFICIENT` meaning `keine Kentnisse`, `Grundkentnisse`, `Fortgeschritten`, `Professionell`

so `VERY_GOOD` and `GOOD` should be removed

File: src/main/webapp/i18n/translations.csv [hunk 2]
```diff
@@ -264,6 +267,7 @@ job-detail,job-detail.language.ku,Kurdisch,Kurdisch,Kurdisch,Kurdisch
 job-detail,job-detail.language.level.BASIC,Grundkentnisse,Grundkentnisse,Grundkentnisse,Grundkentnisse
 job-detail,job-detail.language.level.GOOD,Gut,Gut,Gut,Gut
```
Comment (michael-wirth) [inline src/main/webapp/i18n/translations.csv]: delete

File: src/main/webapp/i18n/translations.csv [hunk 3]
```diff
@@ -264,6 +267,7 @@ job-detail,job-detail.language.ku,Kurdisch,Kurdisch,Kurdisch,Kurdisch
...
 job-detail,job-detail.language.level.NONE,keine Kentnisse,keine Kentnisse,keine Kentnisse,keine Kentnisse
+job-detail,job-detail.language.level.PROFICIENT,Professionell,Proficient,Proficient,Proficient
 job-detail,job-detail.language.level.VERY_GOOD,Sehr Gut,Sehr Gut,Sehr Gut,Sehr Gut
```
Comment (michael-wirth) [inline src/main/webapp/i18n/translations.csv]: delete job-detail.language.level.VERY_GOOD

Comment (michael-wirth) [reply src/main/webapp/i18n/translations.csv]: add job-detail.language.level.INTERMEDIATE

**Pull request merged**